linux_os/guide/services/ssh/file_permissions_sshd_config/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+title: 'Verify Permissions on SSH Server config file'
+
+description: |-
+    {{{ describe_file_permissions(file="/etc/ssh/sshd_config", perms="0600") }}}
+
+rationale: |-
+    The /etc/ssh/sshd_config file needs to be protected from unauthorized changes by
+    non-privileged users.
```

Comment:
Please change the `rationale` to the following:

```
rationale: |-
    Service configuration files enable or disable features of their respective services that if configured incorrectly
    can lead to insecure and vulnerable configurations. Therefore, service configuration file permissions should be
    configured to prevent unauthorized changes or access.
```